### DISCUSSION TITLE
Looped autoplay video shortcode

### DISCUSSION THREAD

**Discussion by cgoo4** [quarto-dev/quarto-cli#3654] [Feature Requests]
Please could the video shortcode support a _looped muted autoplay_ in Quarto's `html` format?

`{{< video somevideo.mp4 autoplay muted loop >}}`

I tried the above with an `.mp4` in Safari.

**Comment by mcanouil** [comment]
Related in some aspects to #3210

**Reply by mcanouil** [reply]
Closing this in favour of <https://github.com/quarto-dev/quarto-cli/issues/3210>.
